src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -171,9 +165,18 @@ public void resetEncoders() {
   /**
    * resets the gyro and the odometry
    */
-  public void resetGyro() {
+  private void resetGyro() {
     m_gyro.reset();
-    resetOdometry();
+  }
+
+  public void resetDriveTrain() {
+    resetDriveTrain(0.0, 0.0, 0.0);
+  }
+
+  public void resetDriveTrain(double xPos, double yPos, double angle) {
+    resetEncoders();
+    resetGyro();
+    resetOdometry(xPos, yPos, angle);
```

Comment:
Seems a bit overkill to have all these private one-line functions - Why not just move the functionality into this function? It's also a bit dangerous, since it's still possible to reset one of them individually (Although only from within the subsystem)

Reply:
Resolved in 39a8ae883c904573ca4ff47c598cede92706d725